.infra/workers.ts

```diff
@@ -215,4 +215,8 @@ export const workers: Worker[] = [
     topic: 'analytics-api.v1.experiment-allocated',
     subscription: 'api.experiment-allocated',
   },
+  {
+    topic: 'api.v1.source-squad-created',
+    subscription: 'api.source-squad-created',
```

Comment:
My suggestion to make it a bit more explicit

```suggestion
    subscription: 'api.source-squad-created-mailing',
```